docs/howto/system_testing.md

```diff
@@ -0,0 +1,140 @@
+```
+
+Here, `SERVICE_LOGS_DIR` is a special keyword. It is something that we will need later.
+
+### Dataset-level configuration
+
+Next, we must define configuration for each data stream that we want to system test.
+
+```
+<package root>/
+  data_stream/
+    <data stream>/
+      _dev/
+        test/
+          system/
+            config.yml
+```
+
+The `config.yml` file allows you define values for package and data stream-level variables. For example, the `apache/access` data stream's `config.yml` is shown below.
+
+```
+vars: ~
+data_stream:
+  vars:
+    paths:
+      - "{{SERVICE_LOGS_DIR}}/access.log*"
+```
+
+The top-level `vars` field corresponds to package-level variables defined in the `apache` package's `manifest.yml` file. In the above example we don't override any of these package-level variables, so their default values, as specified in the `apache` package's `manifest.yml` file are used.
+
+The `data_stream.vars` field corresponds to data stream-level variables for the current data stream (`apache/access` in the above example). In the above example we override the `paths` variable. All other variables are populated with their default values, as specified in the `apache/access` data stream's `manifest.yml` file.
+
+Notice the use of the `{{SERVICE_LOGS_DIR}}` placeholder. This corresponds to the `${SERVICE_LOGS_DIR}` variable we saw in the `docker-compose.yml` file earlier. In the above example, the net effect is as if the `/usr/local/apache2/logs/access.log*` files located inside the Apache integration service container become available at the same path from Elastic Agent's perspective.
+
+#### Placeholders
+
+The `SERVICE_LOGS_DIR` placeholder is not the only one available for use in a data stream's `config.yml` file. The complete list of available placeholders is shown below.
+
+| Placeholder name | Data type | Description |
+| --- | --- | --- |
+| `Hostname`| string | Addressable host name of the integration service. |
+| `Ports` | []int | Array of addressable ports the integration service is listening on. |
+| `Port` | int | Alias for `Ports[0]`. Provided as a convenience. |
+| `Logs.Folder.Agent` | string | Path to integration service's logs folder, as addressable by the Agent. |
+| `SERVICE_LOGS_DIR` | string | Alias for `Logs.Folder.Agent`. Provided as a convenience. |
+
+Placeholders used in the `config.yml` must be enclosed in `{{` and `}}` delimiters, per Handlebars syntax.
+
+## Running a system test
+
+Once the two levels of configurations are defined as described in the previous section, you are ready to run system tests for a package's data streams.
+
+First you must deploy the Elastic Stack.
+
+```
+elastic-package stack up -d
+```
+
+For a complete listing of options available for this command, run `elastic-package stack up -h` or `elastic-package help stack up`.
+
+Next you must set environment variables needed for further `elastic-package` commands.
+
+```
+eval $(elastic-package stack shellinit)
```

Comment:
or:

```
`elastic-package stack shellinit`
```

Reply:
I believe backticks is now considered legacy syntax. According to https://tldp.org/LDP/abs/html/commandsub.html:

> The $(...) form has superseded backticks for command substitution.

That being said, I just realized the `eval` is redundant. I will remove it.